roles/micado_master/start/templates/micado/micado-manifest.yml.j2

```diff
@@ -309,6 +311,9 @@ spec:
       - name: submitter-out
         hostPath:
           path: /var/lib/micado/toscasubmitter/output
+      - name: wrapper-file
```

Comment:
Is it possible to mount just the file? What if there are other binaries in `/usr/local/sbin` on the host?

Reply:
The file alone can be mounted. I will check it.